pkg/storage/system_intake.go

```diff
@@ -45,6 +45,12 @@ func (s *Store) CreateSystemIntake(ctx context.Context, intake *models.SystemInt
 			existing_contract,
 			cost_increase,
 			cost_increase_amount,
+			contractor,
```

Comment:
To consider: are these fields ever really expected to be populated right at `SystemIntake` creation time, or would they only ever actually be populated on an `Update...`?